README.md

```diff
@@ -30,145 +30,144 @@ Styleguide for low-fidelity paper prototyping and wireframing.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/47.PNG)
 
 ### 5. Red elements ❤️ Modifiers
 
-5.1. _MUST_. Use the **red** color to indicate special behaviors of features.
+5.1. _MUST_ use **red** color to indicate features' special behaviors.
 
-5.2. _MUST_. Mark read-only, blocked and disabled controls with an "block" sign.
+5.2. _MUST_ use "blocked" signs (`🚫`) for read-only, blocked or disabled controls.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/52.PNG)
 
-5.3. _MUST_. Mark elements and controls affected by user roles (privileges elevation) with a (`△`). Elements marked both with 5.2 and this marking will be considered not read-only if the role is satisfied. Indicate more details of the privilege and control behavior in the Table of Features.
+5.3. _MUST_ use triangles (`△`) for elements and controls affected by user roles (privileges elevation). Elements marked with both "blocked" sign and triangle will be considered not read-only if the role is satisfied. Describe further details of privileges and control behaviors in the features table.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/53.PNG)
 
-5.4. _MUST_. Mark elements and controls that contains or triggers AJAX loaders with an horizontal three-dot.
+5.4. _MUST_ use horizontal ellipsis (`...`) for elements or controls that contain or trigger AJAX loaders.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/54.PNG)
 
-5.5. _MUST_. Mark elements and controls that are repeatable by some iteration with a vertical three-dot. Indicate more details of the iteration in the Table of Features.
+5.5. _MUST_ use vertical ellipsis (`⋮`) for elements or controls that repeat by some iteration with a vertical three-dot. Indicate more details of the iteration in the Table of Features.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/55.PNG)
 
 ### 6. Green elements 💚 Controls
 
-6.1. _MUST_. Draw **input/text boxes** with just an horizontal line. DO NOT draw inputs as a rectangle, as they may be confused with a button.
+6.1. _MUST_ draw **input/text boxes** with just a horizontal line. DO NOT draw inputs as rectangles, as they may be confused with buttons.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/61.PNG)
 
-6.2. _MUST_. Draw **buttons** as rectangles, including **image buttons**. For buttons that have a cancel/abort/close behavior include a times character (`×`) aligned to center.
+6.2. _MUST_ draw **buttons** as rectangles, including **image buttons**. For buttons that have a cancel/abort/close behavior include a times character (`×`) aligned to its center.
 
 ![](https://raw.githubusercontent.com/NOALVO/wireguide/master/img/62.PNG)
 
-6.3. _MUST_. Draw **checkboxes** as squares with a checkmark inside.
+6.3. _MUST_ draw **checkboxes** as squares with a chec kmark inside (`☑`).
```

Comment:
`checkmark`

Reply:
There was a typo, but it's `check mark`, though.
`Checkmark` is a verb, the action of checking a mark.
http://www.dictionary.com/browse/checkmark